extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -445,12 +467,19 @@ export class AmpAutocomplete extends AMP.BaseElement {
   }
 
   /**
-   * Handle showing or hiding results on user focus/blur.
+   * Disables or re-enables the browser autofill on the autocomplete input.
+   * Then handles showing or hiding results on user focus/blur.
    * @param {boolean} display
    * @return {!Promise}
    * @private
    */
   toggleResultsHandler_(display) {
+    // Set/reset "autocomplete" attribute on the <form> ancestor.
+    const updatedAttribute = display ? 'off' :
+      this.formAncestorAutocompleteAttr_;
+    this.formAncestor_.setAttribute('autocomplete', updatedAttribute);
```

Comment:
Does this set the attribute if the form doesn't have it originally?

Reply:
Yes, this sets it to the implied default so it doesn't change functionality. I see your point that it would probably be better to stay as true to the developer's intent as possible, especially when performing magic like this.